python/GangaLHCb/Lib/GaudiRunApp/GaudiRunApp.py

```diff
@@ -0,0 +1,250 @@
+
+from Ganga.GPIDev.Adapters.IPrepareApp import IPrepareApp
+from Ganga.GPIDev.Schema import Schema, Version, SimpleItem, GangaFileItem
+
+from Ganga.GPIDev.Lib.File.File import ShareDir
+from Ganga.Core import ApplicationConfigurationError, GangaException
+
+from Ganga.Utility.logging import getLogger
+
+from Ganga.GPIDev.Base.Proxy import getName
+
+from os import rename, path, makedirs, chdir
+import shutil
+from Ganga.Utility.files import expandfilename
+
+from Ganga.GPIDev.Lib.File.LocalFile import LocalFile
+from GangaDirac.Lib.Files.DiracFile import DiracFile
+
+from GangaDirac.Lib.Backends.DiracBase import DiracBase
+
+import tempfile
+import time
+import subprocess
+
+logger = getLogger()
+
+def _exec_cmd(cmd, cwdir):
```

Comment:
Yes for now, I'll review in the future changing this, I'm aware this is not ideal but for now I wanted full access to everything for testing/development unless you know a 1 line which I can replace this with?